styx-scheduler-service/src/main/java/com/spotify/styx/StyxScheduler.java

```diff
@@ -584,10 +564,6 @@ private void setupMetrics(
       StateManager stateManager,
       BiConsumer<Optional<Workflow>, Optional<Workflow>> workflowConsumer) {
     return (workflow) -> {
-      stats.registerActiveStatesMetric(
```

Comment:
I think this metric can still be useful (also for user facing metrics), any specific reason it was deleted?

Reply:
Didn't see that we were using it anywhere. We can re-add it. Preferably as a cheap datastore query or something to avoid excessive per-active instance entity reads just for the sake of this metric.

Reply:
This already is a datastore query and it is executed per workflow rather than workflow instance. Impact on storage should be acceptable. 
However, for this best-effort situations that have low requirements on states consistency we could implement a "slow" caching in memory. For now I am just going to add back the metric.